docs/getting-started-with-scalardb.md

```diff
@@ -48,29 +48,12 @@ You can create a JSON file `emoney-storage.json` with the JSON below.
 }
 ```
 
-To apply the schema, download the scalar schema standalone loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
+To apply the schema, download the **scalardb-schema-loader** that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
 
-For Cassandra
 ```
-$ java -jar scalar-schema-standalone-<version>.jar --cassandra -h localhost -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f emoney-storage.json -R 1
-```
-
-For Cosmos DB
-```
-$ java -jar scalar-schema-standalone-<version>.jar --cosmos -h <COSMOS_DB_ACCOUNT_URI> -p <COSMOS_DB_KEY> -f emoney-storage.json
-```
-  - `<COSMOS_DB_KEY>` you can use a primary key or a secondary key.
-
-For DynamoDB
-```
-$ java -jar scalar-schema-standalone-<version>.jar --dynamo -u <AWS_ACCESS_KEY_ID> -p <AWS_ACCESS_SECRET_KEY> --region <REGION> -f emoney-storage.json
-```
-  - `<REGION>` should be a string to specify an AWS region like `ap-northeast-1`.
-
-For JDBC databases
-```
-$ java -jar scalar-schema-standalone-<version>.jar --jdbc -j <JDBC_URL> -u <USERNAME> -p <PASSWORD> -f emoney-storage.json
+$ java -jar scalardb-schema-loader-<version>.jar --config <path_to_database.properties_file> -f emoney-storage.json
 ```
+- The **database.properties** file contains the configurations for Scalar DB which was mentioned in **Getting Started with Scalar DB on X** documents.
```

Comment:
I think `Getting Started with Scalar DB on X` should be linked.

Reply:
I updated it in [commit](https://github.com/scalar-labs/scalardb/pull/352/commits/4d0a25733cc01baf98adfd8c8a1b41b3815fa206)